### DISCUSSION TITLE
Assistance Needed in Achieving a Specific Layout in Obsidian

### DISCUSSION THREAD

**Discussion by JamieAUS** [TfTHacker/DashboardPlusPlus#27] [Q&A]
I'm trying to configure my Obsidian dashboard to match a specific layout as per some instructions I've been following, but I'm struggling to get it right. Despite my efforts, my current layout doesn't align with the examples shown in the instructions.
<img width="719" alt="Screenshot 2024-05-13 at 8 26 24 am" src="https://github.com/TfTHacker/DashboardPlusPlus/assets/42209421/ec9312ab-5db7-4e4b-95c2-f0f91c2a477a">

![Screenshot 2024-05-13 at 8 26 15 am](https://github.com/TfTHacker/DashboardPlusPlus/assets/42209421/25a8aeae-77bf-4e1a-aa39-151403d8efc9)

![Screenshot 2024-05-13 at 10 42 06 am](https://github.com/TfTHacker/DashboardPlusPlus/assets/42209421/9d983b2f-ef4d-42eb-bc27-391882d93a19)

writing mode and reading mode all same.

**Comment by TfTHacker** [comment]
Long over due to ask, did you ever resolve this?
